File: BookTracker.Api/Models/Book.cs
namespace BookTracker.Api.Models;

public enum BookStatus
{
    WantToRead,
    Reading,
    Completed,
    DidNotFinish
}

public class Book
{
    public int Id { get; set; }

    public string Title { get; set; } = "";

    public string Author { get; set; } = "";

    public BookStatus Status { get; set; }
}

File: book-tracker-app/src/app/models/book.ts
export enum BookStatus {
    WantToRead = "WantToRead",
    Reading = "Reading",
    Completed = "Completed",
    DidNotFinish = "DidNotFinish"
}


export interface Book {
 id: number,
 title: string;
 author: string;
 status: BookStatus; 
}